assets/javascript/game2.js

```diff
@@ -1,141 +1,69 @@
 $( document ).ready(function() {
     //console.log( "ready!" );
 
-//global variables
-  let wins = 0;
-  let losses = 0;
-  let numberToGuess;
-  let playerTotal = 0;
-  let value;
+    //global variables
+    const reset = init;
+    let wins = 0;
+    let losses = 0;
+    let numberToGuess;
+    let playerTotal;
 
-  
-function reset (){
-    $(".crystal").each(function(){
-        numberToGuess = Math.floor(Math.random() * 101 + 19);
-        //console.log(value);
-        {
-            $(this).attr({
-                "data-random":numberToGuess
-            });
-        }
-        //reset player score
-        playerTotal = 0;
-
-        //update total score
-        $("#totalScore").text(playerTotal);
-
-        //update random number to guess
-        $("#randomNumber").text(numberToGuess);
-    })
-    
-
-
-}
-reset()
+    init();
 
+    function generateNumberToGuess() {
+        return Math.floor(Math.random() * 101 + 19);
+    }
 
-//game start
-function gameStart(){
-    $(".crystal").click(function(){
-       value = Math.floor(Math.random() * 11 + 1);
```

Comment:
this is what caused the crystals to have different values each time you clicked on them. you're assigning whatever comes from 
````js
Math.floor(Math.random() * 11 + 1);
````
to `value` **each time** the button is clicked. You need to set the crystal value outside the scope of the click handler. This will make a lot more sense when you go over closure.